src/System.Net.Http/ref/System.Net.Http.cs

```diff
@@ -510,23 +510,23 @@ public partial class NameValueHeaderValue : System.ICloneable
         public NameValueHeaderValue(string name, string value) { }
         public string Name { get { throw null; } }
         public string Value { get { throw null; } set { } }
-        object System.ICloneable.Clone() { throw null; }
         public override bool Equals(object obj) { throw null; }
         public override int GetHashCode() { throw null; }
         public static System.Net.Http.Headers.NameValueHeaderValue Parse(string input) { throw null; }
+        object System.ICloneable.Clone() { throw null; }
         public override string ToString() { throw null; }
         public static bool TryParse(string input, out System.Net.Http.Headers.NameValueHeaderValue parsedValue) { throw null; }
     }
     public partial class NameValueWithParametersHeaderValue : System.Net.Http.Headers.NameValueHeaderValue, System.ICloneable
     {
-        protected NameValueWithParametersHeaderValue(System.Net.Http.Headers.NameValueWithParametersHeaderValue source) : base(default(string)) { }
-        public NameValueWithParametersHeaderValue(string name) : base(default(string)) { }
-        public NameValueWithParametersHeaderValue(string name, string value) : base(default(string)) { }
+        protected NameValueWithParametersHeaderValue(System.Net.Http.Headers.NameValueWithParametersHeaderValue source) : base (default(System.Net.Http.Headers.NameValueHeaderValue)) { }
+        public NameValueWithParametersHeaderValue(string name) : base (default(System.Net.Http.Headers.NameValueHeaderValue)) { }
+        public NameValueWithParametersHeaderValue(string name, string value) : base (default(System.Net.Http.Headers.NameValueHeaderValue)) { }
```

Comment:
These two overloads are wrong now.  They should be still using 'string' passing into base().

Reply:
The base-ctor call isn't actually part of the metadata; the generator just finds something on the base class that it can legally call to make the ref.cs compile.